lib/code_corps/auth/bearer_auth_pipeline.ex

```diff
@@ -0,0 +1,8 @@
+defmodule CodeCorps.Auth.BearerAuthPipeline do
+  use Guardian.Plug.Pipeline, otp_app: :code_corps,
+                              module: CodeCorps.Guardian,
+                              error_handler: CodeCorps.Auth.ErrorHandler
+
+  plug Guardian.Plug.VerifyHeader, realm: "Bearer"
+  plug Guardian.Plug.LoadResource, allow_blank: true
```

Comment:
This didn't need to be `allow_blank: true` before but appears to need to be now.

Reply:
Looks like this is accurate and will need to remain this way.